twingate/client.go

```diff
@@ -1,3 +1,4 @@
+// Package twingate is a Terraform provider for Twingate platform
```

Comment:
@ekampf is this comment ok for the package level?

Reply:
@vmanilo  Why do we need it?

Reply:
there was some linter warning about missing comment for the package

Reply:
you can see it here https://github.com/Twingate/terraform-provider-twingate/actions/runs/3022789437/jobs/4862295618#step:9:169

```
Error: twingate/client.go:1:1: package-comments: should have a package comment (revive)
```